command/agent/alicloud_end_to_end_test.go

```diff
@@ -29,14 +27,11 @@ import (
 )
 
 const (
-	envVarRunAccTests = "VAULT_ACC"
-	envVarAccessKey   = "ALICLOUD_TEST_ACCESS_KEY"
-	envVarSecretKey   = "ALICLOUD_TEST_SECRET_KEY"
-	envVarRoleArn     = "ALICLOUD_TEST_ROLE_ARN"
+	envVarAlicloudAccessKey = "ALICLOUD_TEST_ACCESS_KEY"
```

Comment:
Did you intend to have the alicloud changes in this commit? Since it's a test I think it's OK, just making sure though.